.travis.yml

```diff
@@ -16,7 +16,9 @@ matrix:
 before_script:
   - composer install
 
-script: phpunit
+script:
+  - phpunit
+  - phpcs --standard=PSR2
```

Comment:
https://travis-ci.org/zeeshanu/git-profile/jobs/161301402

Could you please change these commands to `./vendor/bin/phpunit` and `./vendor/bin/phpcs --standard=PSR2`